build.include

```diff
@@ -104,11 +109,20 @@ publishImages() {
     docker tag "${image}:${IMAGE_TAG}" "${REGISTRY}/${image}:${IMAGE_TAG}"
     echo y | docker push "${REGISTRY}/${image}:${IMAGE_TAG}"
     if [[ -n "${THEIA_DOCKER_IMAGE_VERSION}" ]]; then
-      echo "Publishing ${image}:${THEIA_DOCKER_IMAGE_VERSION}..."
-      docker tag "${image}:${IMAGE_TAG}" "${REGISTRY}/${image}:${THEIA_DOCKER_IMAGE_VERSION}"
-      echo y | docker push "${REGISTRY}/${image}:${THEIA_DOCKER_IMAGE_VERSION}"
+      if [[ -n "${ARCH}" ]]; then
+         echo "Publishing ${image}:${THEIA_DOCKER_IMAGE_VERSION}-${TRAVIS_TAG}-${ARCH}..."
+         docker tag "${image}:${IMAGE_TAG}" "${REGISTRY}/${image}:${THEIA_DOCKER_IMAGE_VERSION}-${TRAVIS_TAG}-${ARCH}"
+         echo y | docker push "${REGISTRY}/${image}:${THEIA_DOCKER_IMAGE_VERSION}-${TRAVIS_TAG}-${ARCH}"
+      else
+         echo "Publishing ${image}:${THEIA_DOCKER_IMAGE_VERSION}..."
+         docker tag "${image}:${IMAGE_TAG}" "${REGISTRY}/${image}:${THEIA_DOCKER_IMAGE_VERSION}"
+         echo y | docker push "${REGISTRY}/${image}:${THEIA_DOCKER_IMAGE_VERSION}"
+      fi
     else
       SHORT_SHA=$(git rev-parse --short HEAD)
+      if [[ -n "${ARCH}" ]]; then
```

Comment:
we may not build sha1 builds for travis

Reply:
Okay, will remove sha1 builds from Travis.

Reply:
why don't we want sha1-tagged builds ? Is it because they're intermediate builds that make up the larger :latest and :7.yy.z builds, so they're not useful?

Reply:
I think the question is more that for travis we would need to suffix it then we would need a specific case 
like SHA1_SUFFIX in build system to append a specific tag to the digest.

but then, is it really important that travis builds are not only rolling tags as we mainly want to use these images and 
switch to this build system (and then sha1 will be enabled)

I'm perfectly fine by having SHA1_SUFFIX default to empty and travis setting 'travis' for this value or just 'SKIP_DIGEST_PUBLICATION=true' flag (or both)